x/oracle/types/params.go

```diff
@@ -21,6 +21,7 @@ const (
 	DefaultSamplingTryCount        = uint64(3)
 	DefaultOracleRewardPercentage  = uint64(70)
 	DefaultInactivePenaltyDuration = uint64(10 * time.Minute)
+	DefaultIBCRequestEnabled       = true
```

Comment:
Please set this param to false in `migrate.go` too

Reply:
Like we do on `ibcTransferGenesis.Params.ReceiveEnabled`

Reply:
Oh, OK. Fixed.